app/controllers/api/pods_controller.rb

```diff
@@ -7,6 +7,8 @@
 module Pod
   module TrunkApp
     class PodsController < APIController
+      MINIMUM_COCOAPODS_VERSION = Version.new(ENV['TRUNK_MINIMUM_COCOAPODS_VERSION'] || '0.36.0')
```

Comment:
Since 0.36.0 has a known bug with assets, it may be too optimistic that this is 0.36 and not 0.35 at the moment.